lib/vsc/__init__.py

```diff
@@ -1,38 +1,3 @@
-#!/usr/bin/env python
-##
-# Copyright 2011-2012 Ghent University
-# Copyright 2011-2012 Jens Timmerman
-#
-# This file is part of VSC-tools,
-# originally created by the HPC team of Ghent University (http://ugent.be/hpc/en),
-# with support of Ghent University (http://ugent.be/hpc),
-# the Flemish Supercomputer Centre (VSC) (https://vscentrum.be/nl/en),
-# the Hercules foundation (http://www.herculesstichting.be/in_English)
-# and the Department of Economy, Science and Innovation (EWI) (http://www.ewi-vlaanderen.be/en).
-#
-# http://github.com/hpcugent/VSC-tools
-#
-# VSC-tools is free software: you can redistribute it and/or modify
-# it under the terms of the GNU General Public License as published by
-# the Free Software Foundation v2.
-#
-# VSC-tools is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
-# GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with VSC-tools. If not, see <http://www.gnu.org/licenses/>.
-##
-"""
-Initialize vsc package.
-the vsc namespace is used in different folders allong the system
-so explicitly declare this is also the vsc namespace
-"""
-from pkgutil import extend_path
-
-# we're not the only ones in this namespace
-__path__ = extend_path(__path__, __name__)  #@ReservedAssignment
-
-# here for backwards compatibility
-from vsc.utils import fancylogger
+# allow other packages to extend this namespace, zip safe setuptools style
+import pkg_resources
+pkg_resources.declare_namespace(__name__)
```

Comment:
well, yes, but it *requires* setuptools (`pkg_resources` is part of setuptools, not Python)